Whiskey/Functions/Invoke-WhiskeyTask.ps1

```diff
@@ -115,6 +115,15 @@ function Invoke-WhiskeyTask
 
     $task = $knownTasks | Where-Object { $_.Name -eq $Name }
 
+    if( -not $task )
+    {
+        $task = $knownTasks | Where-Object { $_.Aliases -contains $Name }

```

Comment:
Should we also check to make sure this resolved to a single task? What if there are 2 more tasks that accidentally share a common alias?

Reply:
Fixed. I also added a test and check for duplicate task names.